lmdeploy/pytorch/check_env/dist.py

```diff
@@ -1,16 +1,20 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 
+from lmdeploy.pytorch.config import DistConfig
+
 from .base import BaseChecker
 
 
 class DistChecker(BaseChecker):
     """check dist environment."""
 
-    def __init__(self, tp: int, dp: int, distributed_executor_backend: str, device_type: str, logger=None):
+    def __init__(self, tp: int, ep: int, dp: int, distributed_executor_backend: str, device_type: str, logger=None):
```

Comment:
Since we only support EP>1 on hopper, we can add cuda arch check in `def check()`